src/menus/utils.ts

```
@@ -0,0 +1,725 @@
+import type {CommandInteractionOptionResolver} from "discord.js";
+import * as Djs from "discord.js";
+import {getUl, t} from "../i18n";
+import {type RoleIn, type TChannel, TIMEOUT, type Translation} from "../interface";
+import {getConfig, getRoleIn, setRoleIn} from "../maps";
+import {resolveChannelsByIds} from "../utils";
+import {RoleInMode, RoleInPaginationState, roleInStates, type TrackedItems} from "./interfaces";
```

Biome (2.1.2)

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)